config/accounts_test.go

```diff
@@ -824,3 +824,40 @@ func TestModulesGetConfig(t *testing.T) {
 		})
 	}
 }
+
+func TestAccountPriceFloorsValidate(t *testing.T) {
+
+	tests := []struct {
+		description string
+		pf          *AccountPriceFloors
+		want        []error
+		giveErrs    []error
```

Comment:
Nitpick: Is this field needed? I see it's not altered by any test case, and you could maybe just hardcode an error array on line 859 when calling `validate()`.

Reply:
Done